src-plugins/mscAlgorithmPaint/mscAlgorithmPaintToolBox.cpp

```diff
@@ -298,7 +294,7 @@ AlgorithmPaintToolBox::AlgorithmPaintToolBox(QWidget *parent ) :
     layout->addLayout( brushSizeLayout );
 
     // Magic wand's widgets
-    m_wandUpperThresholdSlider = new medDoubleParameter("Upper Threshold", this);
+    m_wandUpperThresholdSlider = new medIntParameter("Upper Threshold", this);
```

Comment:
Hum, you're talking about the case of volumes of double? Do you think the case will happened? Indeed if you have these kind of volumes...